index.js

```diff
@@ -2,6 +2,36 @@ var fs = require( "fs" );
 var path = require( "path" );
 var spawn = require( "spawnback" );
 
+function unique( count ) {
+	count = count || {};
+	return function( i ) {
```

Comment:
Can you change `i` to `key` so it's more obvious that this isn't an incrementing numeric index.